webviz_subsurface/plugins/_rft_plotter/rft_plotter.py

```diff
@@ -219,22 +212,34 @@ def add_webvizstore(self):
                 ],
             )
         ]
-        if self.csvfile_rft is None:
+        if self.csvfile_rft_ert is None:
             functions.append(
                 (
                     load_csv,
                     [
-                        {
-                            "ensemble_paths": self.ens_paths,
-                            "csv_file": "share/results/tables/rft.csv",
-                        },
                         {
                             "ensemble_paths": self.ens_paths,
                             "csv_file": "share/results/tables/rft_ert.csv",
                         },
                     ],
                 )
             )
+            try:
+                load_csv(self.ens_paths, "share/results/tables/rft.csv")
+                functions.append(
+                    (
+                        load_csv,
+                        [
+                            {
+                                "ensemble_paths": self.ens_paths,
+                                "csv_file": "share/results/tables/rft.csv",
+                            },
+                        ],
+                    )
+                )
+            except KeyError:
```

Comment:
What would be the source code raising the `KeyError` when that happens?

Reply:
If the rft.csv file is not found in any realization fmu.ensemble will raise a KeyError. If this happens we should not append that function to Webviz Store.